agent/src/beerocks/slave/agent_ucc_listener.cpp

```diff
@@ -104,8 +104,19 @@ bool agent_ucc_listener::handle_dev_get_param(std::unordered_map<std::string, st
             return false;
         }
         auto ruid = tlvf::mac_to_string(std::strtoull(params["ruid"].c_str(), nullptr, 16));
-        // We use MAC address as Radio UID, so we can just return it here.
-        value = ruid;
+        if (params.find("ssid") == params.end()) {
+            // No ssid was given, return the radio MAC.
+            // We use MAC address as Radio UID, so we can just return it here.
+            value = ruid;
+            return true;
+        }
+        auto ssid = params["ssid"];
```

Comment:
(nitpick) You have the iterator obtained with `find` to directly get the ssid, so there's no need to look it up again